src/layers/XRLayer/xr-layer-vertex.webgl.glsl

```diff
@@ -1,10 +1,10 @@
+#version 300 es
```

Comment:
This just occurred to me... What do you think about appending `#version 300 es` if it's WebGL2 rather than removing it?

Reply:
Done.